### ISSUE TITLE
Being able to choose the MIT license

### ISSUE THREAD

**Issue by leapforleap** [pydanny/cookiecutter-django#1213]
I met @hackebrot during a cookiecutter presentation today and he asked me to raise an issue of this topic I stumbled upon and was asking about:

This project is licensed under the BSD license. The common understanding for possible licenses of derived work is, that it has to have at least the properties of that license. You can find that in a nice diagram with more explanation under https://www.dwheeler.com/essays/floss-license-slide.html.

Thus, being able to choose the MIT license should not be possible in my opinion. It leads to a situation where the resulting code is possibly not properly licensed.

Furthermore the generated project should include the required aspects of the BSD license.

What do you think? I am no lawyer but I believe that this is a topic that should be seriously discussed.

**Comment by jayfk** [comment]
Yep, that's an interesting question. We probably need to summon a couple of open source license experts here.

Any ideas?

**Comment by pydanny** [comment]
From experience, i know that listening and acting on the opinions of self-appointed experts who aren't attorneys on legal matters is often counter productive and dangerous. 

For what it's worth, I don't think this is a serious issue, but if someone who is an attorney with experience on the topic wants to contradict me, then I'm willing to listen.
